cron/rss_reader.py

```diff
@@ -10,11 +10,12 @@ def __init__(self, feed_url):
     def read_feed(self, newer_than):
         feed = feedparser.parse(self.feed_url)
 
+        print(len(feed.entries))
+
         for idx, entry in enumerate(feed.entries):
```

Comment:
We can also drop the `enumerate` call, since we're no longer using `idx` for anything.